### DISCUSSION TITLE
Improving ease of contribution for beginner student contributors 👨‍🎓

### DISCUSSION THREAD

**Discussion by itsmingjie** [meteor/meteor#11618] [Feature Requests]
Dear Meteor maintainers,

We're reaching out on behalf of an ongoing partnership between California State University Monterey Bay's Computing Talent Initiative and CodeDay, a nonprofit organization. We're working on a program for college students who are new to contributing to open-source software, and we'd love to invite them to help closing some "good first issues" in your project!

One of the goals of our program is to get more students to become comfortable with contributing to larger open source projects. A large codebase can be really intimidating to look at at first, so we'd love to invite you to host a short webcast to help students establish a mental modal of your codebase!

If you are interested, our team (cc @SchmitzAndrew) will work with you to create/improve beginner-friendly documentation on how the code is structured, and how to get a local development environment running. We'll also provide primary mentorship to the students. In exchange, we'll need 30-45 minutes of time from a maintainer, and a commitment to be friendly to the students who are opening their first pull request.

I'm open to discussing more below if you have questions! You can also get in touch with me directly at mingjie@codeday.org.

Mingjie, Program @codeday
Berkeley CS'24

If you would not like to participate, please feel free directly close this issue — sorry for the interruption!

**Comment by filipenevola** [comment]
Hi @itsmingjie this is a great initiative and for sure we can help.

We can discuss details here or by email if you prefer filipe@meteor.com.

What do you believe should be the first step? 

We also have a hackathon coming in October with great prizes. It's not focused directly on open-source contributions but they would have contact with the maintainers during the hackathon as well, read more here https://impact.meteor.com/hackathon

And BTW I just checked the codeday.org landing page snd and it's great. Keep it up. 💪

**Comment by itsmingjie** [comment]
Thanks a lot for the kind words @filipenevola! I'm starting an email thread to coordinate the next steps -- mostly, we just need some time from you to record a "tour" through your codebase (or, if desired, a live webcast with our students). Andrew (cc'ed on the email) can help getting things started. 

And thanks for bringing up the hackathon! I'll pass this on to our team as well :D 

I'll post periodic progress updates under this issue for transparency to the community as well.

**Comment by filipenevola** [comment]
Perfect @itsmingjie, I converted the issue to a discussion 😉 

And we are already in touch by email as well.
